components/calculator.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Buttons from './buttons';
+import Display from './display';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
Great job @amklass11 🤝

- [ ] As stated in the project requirements, kindly make this component a functional component.